support-frontend/assets/helpers/forms/existingPaymentMethods/existingPaymentMethods.ts

```diff
@@ -106,9 +106,17 @@ function subscriptionToExplainerPart(
 	subscription: ExistingPaymentMethodSubscription,
 ): string {
 	const activeOrRecentPrefix = subscription.isActive ? 'current' : 'recent';
+	const subscriptionName =
```

Comment:
A switch statement might be more readable? What will we do contributions?

Reply:
Have changed to Switch statement as suggested in the comment.
This function handles contributions including Supporter Plus as well as subscriptions (GW, and Newspaper).For contributions we needed a change in capping in user facing copy.
current Monthly Contribution>>changed to >>current monthly contribution
current Annual Contribution>>changed to >>current annual contribution